### PR TITLE
Creating seperate homepages/ teller authentication to teller's index

### PR DESCRIPTION
**Pull request by mushroomyyy**

### CODE REVIEW

**Review by Jus323 — APPROVED** [review]

**Review by helmyrsln — APPROVED** [review]

File: capstone_db.session.sql
Comment (majulahsingapuri) [inline capstone_db.session.sql:1]: Keep these values commented

File: src/main/java/com/example/capstone3/MyController.java [hunk 1]
```diff
@@ -29,7 +32,21 @@ public String showMain(Model model, Principal principal) {
...
         model.addAttribute("username", principal.getName());
         model.addAttribute("principal", principal);
-        return "index";
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        boolean hasAdminRole = authentication.getAuthorities().stream()
+        .anyMatch(r -> r.getAuthority().equals("ROLE_ADMIN"));
+        boolean hasTellerRole = authentication.getAuthorities().stream()
+        .anyMatch(r -> r.getAuthority().equals("ROLE_TELLER"));
+        System.out.println("has admin role is: " + hasAdminRole);
+        System.out.println("has teller role is: " + hasTellerRole);
```
Comment (majulahsingapuri) [inline src/main/java/com/example/capstone3/MyController.java]: remove print statements

**Review by majulahsingapuri — CHANGES_REQUESTED** [review]
Is the index file still there? you might have to remove the default one then

**Review by mushroomyyy — COMMENTED** [review]
Noted and made the necessary changes

**Review by majulahsingapuri — APPROVED** [review]

**Pull request merged**